Megaparsec/Errors/StreamErrors.lean

```diff
@@ -51,7 +51,7 @@ def toHints (streamPos : Nat) (e : ParseError β E) : Hints β :=
     | ParseError.fancy _ _ => []
     | ParseError.trivial errOffset _ ps =>
         if streamPos == errOffset
-           then (if ps.isEmpty then [] else [ps.keys.toList])
+           then (if ps.isEmpty then [] else [ps.keys.toArray.toList])
```

Comment:
Another place where a direct `.toList` can work

Reply:
Thanks! Forgot about this one.

Turns out there's a coercion of Keys to list via `toList`.

Unfortunately unification isn't smart enough to figure out it can be used in the previous cases, but it can be used here.